Web/starter_project_vanguard/components/profile/AccountSetting.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react'
+import { AiFillEyeInvisible, AiFillEye } from 'react-icons/ai'
+
+const AccountSetting: React.FC = () => {
+  const [showCurrPassword, setShowCurrPassword] = useState<boolean>(false)
+  const [showNewPassword, setShowNewPassword] = useState<boolean>(false)
+  const [showConfirmPassword, setShowConfirmPassword] = useState<boolean>(false)
+
+  return (
+    <div className="mt-5 flex flex-col justify-center gap-2 px-auto">
+      <div className="flex justify-between py-3">
+        <div className="font-montserrat ">
+          <h1 className="font-semibold text-xl">Manage your account</h1>
+          <p className="text-primary-text">
+            you can change your password here.
+          </p>
+        </div>
+        <button className="bg-primary text-white rounded px-3 h-9 my-auto mr-8">
+          Save Changes
+        </button>
+      </div>
+      <hr />
+      <form className="mx-auto font-poppins font-semibold text-tertiary-text pt-5 flex flex-col gap-4">
```

Comment:
I think a flex or a grid might get you better-aligned results
but we can do much UI changes in a different PR